printers@linux-man/files/printers@linux-man/po/Nl.po

```diff
@@ -3,100 +3,100 @@
 # This file is distributed under the same license as the PACKAGE package.
```

Comment:
It looks like you renamed the base template POT file to Nl.po. This will break translation maintenance, please copy instead.